subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi.h

```diff
@@ -34,28 +34,54 @@ static inline void hal_radio_enable_on_tick_ppi_config_and_enable(uint8_t trx)
 	/* No need to configure anything for the pre-programmed channels.
 	 * Just enable and disable them accordingly.
 	 */
-	nrf_ppi_channels_disable(
-		NRF_PPI,
-		trx ? BIT(HAL_RADIO_ENABLE_RX_ON_TICK_PPI)
-		    : BIT(HAL_RADIO_ENABLE_TX_ON_TICK_PPI));
-	nrf_ppi_channels_enable(
-		NRF_PPI,
-		trx ? BIT(HAL_RADIO_ENABLE_TX_ON_TICK_PPI)
-		    : BIT(HAL_RADIO_ENABLE_RX_ON_TICK_PPI));
+	if (trx) {
```

Comment:
Update the comment above since you are not disabling the channels

Reply:
`nrf_ppi_channels_enable` name is misleading, it takes bitmask that enables (set) and disables (unset).